src/sentry/static/sentry/app/routes.jsx

```diff
@@ -615,6 +615,12 @@ function routes() {
           import(/*webpackChunkName:"AcceptProjectTransfer"*/ 'app/views/acceptProjectTransfer')}
         component={errorHandler(LazyLoad)}
       />
+      <Route
+        path="/integration/:installationId"
```

Comment:
Do we want to reserve this URL route, seems like we probably want this to actually end up somewhere under settings, otherwise we need to reserve `integration`